resources/views/auth/passwords/reset.blade.php

```diff
@@ -15,12 +15,16 @@
                         <h4>{{ trans('auth.validation.issues') }}</h4>
                         <ul class="list -compacted">
                             @foreach ($errors->all() as $error)
-                                <li>{{ $error }}</li>
+                                <li>
+                                    {{ $error }} 
+                                    @if (strpos($error, 'create-password'))
+                                        <a href="/password/reset">Resent email to create/reset password</a>
+                                    @endif
+                                </li>
                             @endforeach
                         </ul>
                     </div>
-                @endif
-
+                @endif              
```

Comment:
I think we should keep the extra line below this `@endif` since it makes it easier to read!
```suggestion
                @endif
                              
```